frame/tips/src/lib.rs

```diff
@@ -408,6 +410,35 @@ decl_module! {
 			Tips::<T>::remove(hash);
 			Self::payout_tip(hash, tip);
 		}
+
+		/// Remove and slash an already-open tip.
```

Comment:
Consider adding a sentence to the module doc about this as well :)

Reply:
Good one, thanks @kianenigma, will update the module docs